app/javascript/src/app/controllers/campaign_form_controller.js

```diff
@@ -40,16 +39,12 @@ export default class extends Controller {
       credentials: 'same-origin',
       headers: { 'X-CSRF_Token': Rails.csrfToken() }
     })
-      .then(
-        (response => {
-          return response.text()
-        }).bind(this)
-      )
-      .then(
-        (html => {
-          this.setCreativeOptions(html)
-        }).bind(this)
-      )
+      .then(response => {
+        return response.text()
+      })
+      .then(html => {
+        this.setCreativeOptions(html)
+      })
```

Comment:
Let's be sure to test this form thoroughly prior to merge.

Reply:
Basically everything that ESLint prompted for a `bind` removal.

Reply:
What if we just disable this rule if we don't feel comfortable about it?